core/src/main/resources/lib/hudson/buildHealth.jelly

```diff
@@ -37,36 +37,37 @@ THE SOFTWARE.
+        <x:attribute name="class">jenkins-table__cell--tight jenkins-table__icon healthReport</x:attribute>
+        <x:attribute name="onmouseover">this.className='jenkins-table__cell--tight jenkins-table__icon healthReport hover';return true;
         </x:attribute>
-        <x:attribute name="onmouseout">this.className='healthReport';return true;</x:attribute>
+        <x:attribute name="onmouseout">this.className='jenkins-table__cell--tight jenkins-table__icon healthReport';return true;</x:attribute>
         <j:if test="${buildHealth!=null}">
-            <j:if test="${iconSizeClass == null}">
-              <j:choose>
-                <j:when  test="${iconSize == null}">
-                  <j:set var="iconSizeClass" value="icon-sm"/>
+            <div class="jenkins-table__cell__button-wrapper">
+                <j:if test="${iconSizeClass == null}">
+                  <j:choose>
+                    <j:when  test="${iconSize == null}">
+                      <j:set var="iconSizeClass" value="icon-sm"/>
+                    </j:when>
+                    <j:otherwise>
+                      <j:set var="iconSizeClass" value="${icons.toNormalizedIconSizeClass(iconSize)}"/>
+                    </j:otherwise>
+                  </j:choose>
+                </j:if>
+                <j:choose>
+                <j:when test="${!empty(healthReports)}">
+                    <a class="build-health-link jenkins-table__button" href="${empty(link)?'#':link}" style="${attrs.style}">
+                       <l:svgIcon href="${resURL}/images/${icons.getIconByClassSpec(buildHealth.iconClassName + ' icon-md').url}" ariaHidden="true"/>
+                    </a>
                 </j:when>
-                <j:otherwise>
-                  <j:set var="iconSizeClass" value="${icons.toNormalizedIconSizeClass(iconSize)}"/>
-                </j:otherwise>
-              </j:choose>
-            </j:if>
-            <j:choose>
-              <j:when  test="${!empty(healthReports)}">
-                <a class="build-health-link" href="${empty(link)?'#':link}" 
-                    style="${attrs.style}">
-                    <l:icon class="${buildHealth.iconClassName} ${iconSizeClass}" alt="${buildHealth.score}%"/>
-                </a>
-              </j:when>
-              <j:otherwise>
-                 <l:icon class="${buildHealth.iconClassName} ${iconSizeClass}" alt="${buildHealth.score}%" />
-               </j:otherwise>
-             </j:choose>  
+                    <j:otherwise>
+                        <l:svgIcon href="${resURL}/images/${icons.getIconByClassSpec(buildHealth.iconClassName + ' icon-md').url}" tooltip="${buildHealth.score}%" />
+                    </j:otherwise>
+                </j:choose>
+            </div>
         </j:if>
         <j:if test="${!empty(healthReports)}">
             <div class="jenkins-tooltip healthReportDetails">
-                <table class="bigtable stripped-odd">
+                <table class="jenkins-table">
```

Comment:
Also here: we need to restyle all existing plugins. Isn't there a better alternative without changing the plugins by substituting the existing classes?